app/src/main/java/edu/uc/forbesne/shoppingsidekick/service/ProductService.kt

```diff
@@ -14,11 +14,17 @@ import retrofit2.Response
 /**
  * Makes calls to APIs to get product data
  *
+ * Contains the following methods:
+ *
+ * fetchProductsByDescription, which fetches a specific product by its given description
+ * fetchAllProductsFromOneStore, which fetches all product from a particular store
+ * fetchAllProductsFromTwoStore, which fetches all the product sold by two different stores
+ * fetchAllProductsFromThreeStore, which fetches all the product sold by three different stores
+ *
  */
-
 class ProductService {
 
-    fun fetchProductsByName(productName: String) : MutableLiveData<ArrayList<Product>> {
+    fun fetchProductsByDescription(productName: String) : MutableLiveData<ArrayList<Product>> {
```

Comment:
It looks like it is fetching by name... keep this as is.

Reply:
There is no parameter named "name" or "productName" in the product DTO  and instead, this method fetches the product by its description, that's the reason I changed the name of the method instead of changing the DTO class.